config/datasets/ot_gcp.yaml

```diff
@@ -64,4 +63,5 @@ colocalisation: ${datasets.release_folder}/colocalisation
 study_index: ${datasets.release_folder}/study_index
 variant_index: ${datasets.release_folder}/variant_index
 credible_set: ${datasets.release_folder}/credible_set
+gene_index: ${datasets.release_folder}/gene_index
```

Comment:
I still don't understand why the target dataset is in the output. The ETL does nothing with genes: we are not aggregating information for genes, we are not enriching the gene dataset (as far as I know). All gene related information is either in the v2g or variant annotation dataset. Again, if there would be an actual product downstream that would require gene metadata I would understand the placement.

Reply:
I think we are all thinking the same from slightly different angles. No right or wrong. Whatever we do now is likely to change again

Reply:
It's simply because the gene index is a dependency of V2G.
If it's a big deal, I can delete it after V2G is done.

Reply:
No. Intervals are a static asset, target is not.